src/fides/api/api/v1/endpoints/privacy_preference_endpoints.py

```diff
@@ -74,6 +87,25 @@
 router = APIRouter(tags=["Privacy Preference"], prefix=V1_URL_PREFIX)
 
 
+def get_served_notice_history_id(
```

Comment:
nit: this gets more than the id, right, it returns the whole object? should it be called `get_served_notice_history`?

Reply:
thanks for catching 👍